src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -232,14 +240,25 @@ public void UpdateShownPatchDetails()
         {
             details.Visible = false;
             nothingSelected.Visible = true;
+            unknownPatch.Visible = false;
 
             return;
         }
 
-        details.Visible = true;
-        nothingSelected.Visible = false;
+        if (!SelectedPatch.Explored && !IgnoreFogOfWar)
+        {
+            details.Visible = false;
+            nothingSelected.Visible = false;
+            unknownPatch.Visible = true;
+        }
+        else
+        {
+            details.Visible = true;
+            nothingSelected.Visible = false;
+            unknownPatch.Visible = false;
 
-        UpdatePatchDetails();
+            UpdatePatchDetails();
```

Comment:
I'm worried that not calling this in all circumstances could lead to information not being updated correctly. Is there a particular reason it was moved inside the else block only?

Reply:
This is the only case where actually updating the info is necessary, as the other two times it's either the "select patch" screen or the "unknown patch" screen

Reply:
That was exactly my impression as well, which is why I didn't comment on this specific part of the changes. So this optimizes things a bit with leaving the old data there when the data is not visible anyway, if I understood the changes right.